src/Microsoft.Health.Fhir.Shared.Web/appsettings.json

```diff
@@ -66,6 +66,23 @@
                 "JobHeartbeatTimeoutThreshold": "00:10:00",
                 "JobPollingFrequency": "00:00:10",
                 "QueryDelayIntervalInMilliseconds": 500
+            },
+            "DataConvert": { 
+                "Enabled": true,
+                "ContainerRegistries": [
+                    {
+                        "ContainerRegistryServer": "",
+                        "ContainerRegistryUsername": "",
+                        "ContainerRegistryPassword": ""
+                    }
+                ],
+                "TemplateCollectionOptions": {
+                    "ShortCacheTimeSpan": "00:30:00",
+                    "LongCacheTimeSpan": "01:00:00",
+                    "TemplateCollectionSizeLimitMegabytes": 10
+                },
+                "ContainerRegistryTokenExpiration": "00:30:00",
+                "ProcessTimeoutThreshold": "00:00:30"
```

Comment:
I suggest calling this "OperationTimeout".

Reply:
Thanks. Updated